package.json

```diff
@@ -25,7 +25,8 @@
   },
   "pnpm": {
     "overrides": {
-      "mime": "^3.0.0"
+      "mime": "^3.0.0",
+      "rollup": "^4.18.0"
```

Comment:
Instead of this, another option is to change the tsconfig.json in apps/wing to have this at the end:
```
  "exclude": ["src/**/*.test.ts"]
```

This is a good idea anyways because it avoids including our tests in the final `dist` (I just noticed that 😱)

Reply:
I just tried it locally and it worked for me. I tried to do this vitest update a while ago and wasn't able to fix this, but finally saw this comment https://github.com/rollup/rollup/issues/5199#issuecomment-2100534676

Reply:
Sure, yeah let me try that (also it seems like this override isn't working like I expected)